pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -495,3 +496,51 @@ type FrameworkHandle interface {
 	// VolumeBinder returns the volume binder used by scheduler.
 	VolumeBinder() scheduling.SchedulerVolumeBinder
 }
+
```

Comment:
how about we place it in a separate file named extender.go instead of placing it in interface.go? I am hoping we can keep interface.go for the main framework interface.

Reply:
Done.